src/community/wps-sextante/src/main/java/org/geoserver/wps/sextante/GTDBFIterator.java

```diff
@@ -0,0 +1,53 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
The headers here are wrong, the 2001-2013 Openplans line should be there only in files that were part of GeoServer before 2014. This one was not, it should only have the first line. And the same goes for other files in your pull request.